tools/clang_tools/api_booster/main.cc

```diff
@@ -485,6 +485,7 @@ class ApiBooster : public clang::ast_matchers::MatchFinder::MatchCallback,
       return result;
     } else if (!absl::StartsWith(proto_type_name, "envoy.HotRestart") &&
                !absl::StartsWith(proto_type_name, "envoy.RouterCheckToolSchema") &&
+               !absl::StartsWith(proto_type_name, "envoy.annotations") &&
```

Comment:
Should the annotations be versioned? Orthogonal to this PR but curious,

Reply:
I don't see annotations as part of the wire contract, since management servers shouldn't be relying on them. So, less concerned about versioning them and it's simpler not to have the concern around versioning the versioning annotations.